build.gradle

```diff
@@ -65,7 +65,7 @@ subprojects {
 
   tasks.withType(JavaCompile) {
     'all -processing -rawtypes -serial'.split().each {
-        options.compilerArgs << "-Xlint:${it}"
+        options.compilerArgs << ("-Xlint:${it}" as String)
```

Comment:
> 'leftShift' in 'org.codehaus.groovy.runtime.DefaultGroovyMethods' cannot be applied to '(groovy.lang.GString)' : 68

Well, apparently it can. And the cast looks a bit silly. So I'd vote against this particular change.

I guess, I'd accept `.toString()` if you really want to silence the warning.